libraries/javascript/package.json

```diff
@@ -21,7 +21,7 @@
     "test": "jest",
     "prepublishOnly": "yarn lint",
     "lint:eslint": "eslint --ignore-path .lintignore --ext .js,.jsx,.ts,.tsx src",
-    "lint:prettier": "prettier --ignore-path .lintignore --write src/**.ts",
```

Comment:
I assume this is just the default with new prettier? If so, feel free to merge.

Reply:
My bad i wanted to have the `-c` on this command. Will fix